custom_components/tuya_local/devices/luko_dehumidifier.yaml

```diff
@@ -0,0 +1,66 @@
+name: Luko Dehumidifier
+products:
+  - id: 4w2fzgr62c1oyjsx
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 8
+      name: current_humidity
+      type: integer
+      unit: "%"
+      class: measurement
+      readonly: true
+      range:
+        min: 15
+        max: 100
+    - id: 2
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "dehumidity"
+          value: normal
+    - id: 46
+      name: humidity
+      type: integer
+      unit: "%"
+      range:
+        min: 40
+        max: 80
+      mapping:
+        - step: 5
+secondary_entities:
+  - entity: fan
+    name: Fan
```

Comment:
Does not need a name if it is the same as the default naming for the entity.

Reply:
ack; I didn't see an immediate way to understand that (especially with capitalisation).